lib/capybara-screenshot/saver.rb

```diff
@@ -98,12 +101,13 @@ def display_image
       private
 
       def current_path
+        # the current_path may raise error in selenium
         begin
-          page.current_path.to_s
+          path = page.current_path.to_s
         rescue StandardError => e
           warn "WARN: Screenshot could not be saved. `page.current_path` raised exception: #{e}."
-          ''
         end
+        yield path if path
```

Comment:
Why are you yielding as opposed to just returning a value?  I am not following the logic